website/docs/security/secrets-management/secrets-manager/ascp.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "AWS Secrets and Configuration Provider (ASCP)"
+sidebar_position: 62
+---
+
+If you ran the `prepare-environment` script detailed in a [previous step](./index.md), it has already installed the AWS Secrets and Configuration Provider (ASCP) for the Kubernetes Secrets Store CSI Driver that's required for this lab.
```

Comment:
"If you ran" is not an if, reword to something like "When you ran"

Reply:
😅